src/main/java/com/baidu/hugegraph/structure/auth/Project.java

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright 2017 HugeGraph Authors
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with this
+ * work for additional information regarding copyright ownership. The ASF
+ * licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.baidu.hugegraph.structure.auth;
+
+import java.util.Date;
+import java.util.List;
+
+import com.baidu.hugegraph.structure.constant.HugeType;
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class Project extends AuthElement {
+
+    @JsonProperty("project_name")
+    private String name;
+    @JsonProperty("project_admin_group")
+    private String adminGroup;
+    @JsonProperty("project_op_group")
+    private String opGroup;
+    @JsonProperty("project_graphs")
+    private List<String> graphs;
+    @JsonProperty("project_graph")
+    private String graph;
```

Comment:
"graph" is unused field?

Reply:
Previously only supported single value update, it has been changed to batch update